spidermon/contrib/scrapy/extensions.py

```diff
@@ -171,7 +171,7 @@ def _run_suites(self, spider, suites):
             runner.run(suite, **data)
 
     def _generate_data_for_spider(self, spider):
-        from spidermon.utils.hubstorage import hs
+        from spidermon.utils.zyte import client as hs
```

Comment:
**question** Why define an alias for this import? We are not using it in other places, so `from spidermon.utils.zyte import client` and then `"job": client.job if client.available else None` looks fine to me.